nfts/frontend/src/App.js

```diff
@@ -1,10 +1,28 @@
-import logo from "./logo.svg";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Navbar from "./Components/Navbar/Navbar";
+import { MetaKeep } from "metakeep";
+import { TransferNFT } from "./Components/NFT/TransferNFT";
 import "./App.css";
+import { useMemo, useState } from "react";
 
 function App() {
+  const [env, _] = useState("prod");
```

Comment:
I think you don't need to use state or memo for SDK

Reply:
yeah, removed this

Reply:
WHy are you still using useMemo? Does it not work without it?

Reply:
TypeError: sdk.getConsent is not a function

Reply:
fixed this